### ISSUE TITLE
Add tags to resume

### ISSUE THREAD

**Issue by ss22ever** [resumic/schema#10]
It will be nice to have tags on resume that will help in filtering purposes and applying to varying jobs

**Comment by stp-ip** [comment]
There was quite a bit of discussion on this. 

My thinking currently is to have tags, that get shown as actual tags on themes etc. for the main sections etc., but for filtering purposes we use keywords.

Tags: public
Keywords: private, for filtering

A bit more context in this thread and the comment: https://github.com/jsonresume/resume-schema/issues/203#issuecomment-161626244

Happy to discuss this further.

**Comment by stp-ip** [comment]
@ArmanMazdaee I think this needs a bit more discussion. Let's jump into this in parallel.

A few things that need to be cleared up first:
* split tags and keywords?
* which one is public and private aka used for filtering or showcasing?
* which section/subsection to add this to?
* different filter mechanism such as a similar one to our i18n idea #81?

**Comment by stp-ip** [comment]
```metadata.tags = [{
    "tags": [
      "pro-bono"
      ],
     "attributes": [
       "work['greenpeace']",
     ]
  }, {
    "tags": [
      "volunteering",
      "greenpeace"
      ],
    "attributes": [
      "projects['volunteering party organization']",
      "events['demonstration 123'].talks['saving cats']"
      ]
}]
```

**Comment by stp-ip** [comment]
Maybe we can use references as available in jsonschema.
`$ref": "#/definitions/person"`

**Comment by stp-ip** [comment]
What we currently have is keywords being public.

The next step would be using a filter and tagging syntax within the meta section.
